packages/compartment-mapper/test/zip.test.js

```diff
@@ -0,0 +1,33 @@
+import tape from "tape";
```

Comment:
It wouldn’t be hard, but I would rather test interop against a higher authority than that.

JSZip uses a bunch of reference Zip files as text fixtures for the read side. I have deliberately avoided making this work for the Zip64, *FLATE, non-zip prefix, and incoherent headers cases (for now), but I could certainly add one case for a zip file generated by other means. Validating the writer would I think involve shelling out to `unzip` and cleaning up the mess afterward. Testing the writer side is more interesting against a variety of other tools. They all have quirks.

Reply:
For now, I’ve checked in a known-good zip file of the app test fixture and a test to verify it can be read out. That’s moderately unsatisfying, but maybe good enough for now. A more interesting case would shell out to `zip` and `unzip` but that’s a bit more messy than I want to tackle now.